packages/ui/src/fields/FieldLabel/index.tsx

```diff
@@ -26,14 +28,15 @@ const DefaultFieldLabel: React.FC<GenericLabelProps> = (props) => {
   const { path } = useFieldProps()
   const editDepth = useEditDepth()
   const htmlFor = htmlForFromProps || generateFieldID(path, editDepth, uuid)
-
   const { i18n } = useTranslation()
+  const { code } = useLocale()
 
   if (labelFromProps) {
     return (
       <Element className={`field-label ${unstyled ? 'unstyled' : ''}`} htmlFor={htmlFor}>
         {getTranslation(labelFromProps, i18n)}
         {required && !unstyled && <span className="required">*</span>}
+        {localized && <span className="localized">{code}</span>}
```

Comment:
Looks like were going with en dash, also the code should only be used when there isn't a `locale.label`